bindings/php/src/lib.rs

```diff
@@ -15,14 +15,165 @@
 // specific language governing permissions and limitations
 // under the License.
 
+use std::collections::HashMap;
+use std::str::FromStr;
 use ext_php_rs::prelude::*;
-use opendal::{EntryMode, Metadata};
+use ext_php_rs::{exception::PhpException, zend::ce};
+use ext_php_rs::convert::FromZval;
+use ext_php_rs::flags::DataType;
+use ext_php_rs::types::Zval;
+use ::opendal as od;
 
-#[php_function]
-pub fn debug() -> String {
-    let metadata = Metadata::new(EntryMode::FILE);
+#[php_class(name = "OpenDAL\\Operator")]
```

Comment:
Does php has concept about async operations?

Reply:
From my understanding, I think there are NO async in PHP. If we find out later that we can support asynchronous calls, we can add it in the future PR. For now, let's stay with synchronous calls.